js/interface.js

```diff
@@ -697,6 +559,129 @@ var DynamicLists = (function() {
         }
       });
     },
+    managePermissionList: function(isAddEntryActive, isEditEntryActive, isDeleteEntryActive) {
```

Comment:
Use an `options` object as input parameter please. If someone calls `managePermissionList(true, false, true)` it isn't very clear what the parameters mean.